content/en/security/cloud_security_management/guide/public-accessibility-logic.md

```diff
@@ -14,22 +14,31 @@ Datadog uses a graph processing framework to map relationships between cloud res
 
 For more information on network reachability, see the [AWS documentation][34] and the [AWS Network Reachability Analyser][35]. Currently, the `Is Publicly Accessible` facet is only available for AWS resources.
 
+This page details on a resource-by-resource basis the logic behind public accessibility determination for a resource.
+
+**Note:** Not all resources are able to be classified as publicly accessible or not publicly accessible through only their resource configuration. Such resources are not included here.
+
 ## Resource dependency graph
 
-The following diagram shows how related resources are used to determine whether other resources are publicly accessible. For example, a CloudTrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the Cloud Security Management Misconfigurations Resource relationships graph.
+The following diagrams show how related resources are used to determine whether other resources are publicly accessible. For example, a CloudTrail Trail stored in a public S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the Cloud Security Management Misconfigurations Resource relationships graph.
```

Comment:
```suggestion
The following diagrams show how related resources are used to determine whether other resources are publicly accessible. For example, an AWS CloudTrail Trail stored in a public Amazon S3 bucket is itself publicly accessible. If a resource is publicly accessible because of another resource, the relationship is shown in the Cloud Security Management Misconfigurations resource relationships graph.
```